controllers/users.js

```diff
@@ -0,0 +1,74 @@
+import jwt from 'jsonwebtoken';
+import crypto from 'crypto';
+import sgMail from '@sendgrid/mail';
+import userModels from '../models/';
+const { User } = userModels;
+
+class Users {
+
+	static async resetPassword(req, res) {
+		// check if email exists in the database
+		const userEmail = req.body.email;
+    const result = await User.findAll({
+					  where: {
+					    email: userEmail
+					  }
+					});
+    if(result.length === 0) {
+        return res.status(404).send({
+            status: res.statusCode,
+            message: 'email not found',
+        });
+    } 
+
+    // create a JWT token
+    const token = await jwt.sign({ userEmail }, process.env.SECRET, { expiresIn: '2h' });
+    console.log(token);
+    //send email using sendgrid
+    sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+		const msg = {
+		  to: userEmail,
+		  from: 'info@authorhaven.com',
+		  subject: 'Sending with SendGrid is Fun',
```

Comment:
This looks like a generic template that you put here. It's better if you refactor it based to the purpose of the email, not leaving sendgrid here.